src/lib/input/input-container.ts

```diff
@@ -302,6 +336,12 @@ export class MdInputContainer implements AfterContentInit {
     this._mdInputChild._placeholderChange.subscribe(() => this._validatePlaceholders());
   }
 
+  ngAfterViewInit() {
+    // Avoid animations on load.
+    this._messageAnimationState = 'enter';
+    this._changeDetectorRef.detectChanges();
```

Comment:
This does seem odd, but perhaps I'm missing something about why its needed.

Does this animation not work if you initially set the messageAnimationState to enter?